universe/Ship.cpp

```diff
@@ -134,7 +134,7 @@ void Ship::Copy(TemporaryPtr<const UniverseObject> copied_object, int empire_id)
             this->m_design_id =             copied_ship->m_design_id;
             for (PartMeterMap::const_iterator it = copied_ship->m_part_meters.begin();
                  it != copied_ship->m_part_meters.end(); ++it)
-            { this->m_part_meters[it->first]; }
+            { this->m_part_meters = copied_ship->m_part_meters; }
```

Comment:
bit unsure about this, but can't immediately think of a case when a particular meter should remain hidden when at partial visibility...